specs/src/ante_handler_v3.md

```diff
@@ -3,7 +3,6 @@
 The AnteHandler chains together several decorators to ensure the following criteria are met for app version 3:
 
 - The tx does not contain any messages that are unsupported by the current app version. See `MsgVersioningGateKeeper`.
-- The tx size is not larger than the application's configured versioned constant [MaxTxSize](https://github.com/celestiaorg/celestia-app/blob/8ba82c1b872b7f5686d9bb91b93a0442223d7bb2/pkg/appconsts/v3/app_consts.go#L9).
```

Comment:
We need a separate PR that targets v3.x and removes the max tx size ante handler from that branch

Reply:
https://github.com/celestiaorg/celestia-app/issues/4554

Reply:
[question] are we actually going to do https://github.com/celestiaorg/celestia-app/issues/4554? If not, then why remove this line from the v3 specs?

Reply:
Ohh is this because there is no ante_handler_v4.md yet?

Reply:
Yeah I think this should stay because we're not changing it but perhaps we need a v4 section to add that the capped size is now enforced by consensus